doaj/software-versions.yml

```diff
@@ -9,117 +9,200 @@
       ignore_errors: true
       when: "'ubuntu' in ansible_distribution|lower"
 
+    - name: Set Ubuntu version message
+      set_fact:
+        ubuntu_msg: "Ubuntu Version: {{ ubuntu_version.stdout }}"
+      when: ubuntu_version is defined and 'stdout' in ubuntu_version
+
     - name: Get Python version
       command: python3 --version
       register: python_version
       changed_when: false
       ignore_errors: true
 
+    - name: Set Python version message
+      set_fact:
+        pyhon_msg: "Python Version: {{ python_version.stdout }}"
+      when: python_version is defined and 'stdout' in python_version
+
+    - name: Get Java version
```

Comment:
I get an error here on all servers- is this due to my own Ansible version?

```
TASK [Get Java version] ******************************************************************************
fatal: [doaj-index-1]: FAILED! => {"changed": false, "msg": "Unsupported parameters for (ansible.legacy.command) module: warn. Supported parameters include: _raw_params, _uses_shell, argv, chdir, creates, executable, removes, stdin, stdin_add_newline, strip_empty_ends."}
```

```
steve@hegemone:~/code/cl/sysadmin/doaj$ ansible --version
ansible [core 2.15.1]
  config file = None
  configured module search path = ['/home/steve/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/lib/python3.10/dist-packages/ansible
  ansible collection location = /home/steve/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/local/bin/ansible
  python version = 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] (/usr/bin/python3)
  jinja version = 3.0.3
  libyaml = True
```

Reply:
Now I have changed the code to use shell. Hopefully it should work now. Could be ansible version issue